CMakeLists.txt

```diff
@@ -152,6 +152,8 @@ target_compile_definitions(gsl INTERFACE GSL_THROW_ON_CONTRACT_VIOLATION)
 # RPATH information
 #===============================================================================
 
+include(GNUInstallDirs)
```

Comment:
At first glance, this looks like a GCC-specific command. Consider dropping a comment on here to explain that it's just for the useful variables.

Reply:
Just added a comment to that effect